RecoHI/HiMuonAlgos/python/HiRegitMuonPixelPairStep_cff.py

```diff
@@ -52,7 +52,7 @@
     clusterCheck = "hiRegitMuClusterCheck",
 )
 hiRegitMuPixelPairStepSeeds     = RecoTracker.IterativeTracking.PixelPairStep_cff.pixelPairStepSeeds.clone(
-    seedingHitSets = "hiRegitMuPixelPairStepHitDoublets"
+    seedingHitSets = cms.InputTag("hiRegitMuPixelPairStepHitDoublets")
```

Comment:
@slava77 Without this type specialization the driver no longer runs, since the changes made here:  https://github.com/cms-sw/cmssw/commit/f9cefe2826f8be81c8b15cd79e57319f4c286d71#diff-528a13022ace4a870beca2a390be12c3R143